### PR TITLE
Functional V0.2

### PR DESCRIPTION
**Pull request by PortetWilfried**

### CODE REVIEW

File: documents/functional_specifications/Functional Specification.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+#### 2.4.1 User Side
+
+Typical steps for users:
+
+1. Download the application.
+2. Select the preferred language for the application's text.
+3. Create an account by providing personal information.
+4. Select relevant soft skills via checkboxes.
+5. Access the homepage to view companies currently hiring for the selected soft skills.
+6. Specify the geographical area for job searching.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:96]: The geographical area should be asked before accessing to the home page.

File: documents/functional_specifications/Functional Specification.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+7. If a company expresses interest in the user’s profile, the connection process begins.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:97]: Before that, the job seeker must like the company first, to initiate the match.

File: documents/functional_specifications/Functional Specification.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+#### 2.4.2 Company Side
+
+Typical steps for companies:
+
+1. Download the application on a computer or mobile device.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:103]: Mobile devices are not targeted for the company side for now.

File: documents/functional_specifications/Functional Specification.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+2. Select the preferred language for the application's text.
+3. Create an account by providing company information.
+4. Access the homepage.
+5. Create job offers by specifying the required soft skills and geographical location.
+6. Post job advertisements on the app.
+7. Wait for applications from candidates.
+8. Review anonymous candidate profiles based on soft skills.
+9. Create a match if interested in a candidate's profile.
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:111]: The company doesn't initiate the match. 
The company gives their "preferred soft-skills", and if a job seeker likes a company with which the soft skills are compatible, the match appears. 
The company decides to send a message to the job seeker afterwards.

**Review by NanaChocolat — APPROVED** [review]
Just the "adopte un candidat" should be in English ligne 55, you can continue.

File: documents/functional_specifications/Functional Specification.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+### 6.1 Description
+
+- **Matching Function:** Matches candidates with job postings based on selected soft skills.
+- **Profile Management:** Allows users to create and manage their profiles, including updating personal information and soft skills.
+- **Job Posting Management:** Enables companies to create, edit, and manage job postings.
+
+### 6.2 Inputs
+
+- **Candidate Inputs:**
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:140]: The Candidate also provides his location.

File: documents/functional_specifications/Functional Specification.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+- First screen :
+  - Choose Your Language
+
+## 8. Hardware Interfaces
+
+- To use the "Adopt a Candidate" app, users looking for a job must access the app via their smartphone by downloading it from the Apple Store or Play Store. The app will be optimized to work on the following devices:
+  - Smartphones with iOS 12 or later.
+  - Smartphones with Android 8.0 (Oreo) or later.
+  - A stable Internet connection is required to access all the features of the application.
+- In a future release, users will also be able to use the app on their personal computers. The minimum specifications laid down are:
```
Comment (EnzoGuillouche) [inline documents/functional_specifications/Functional Specification.md:178]: When you use the term "users", I cannot understand if you're talking about jobseekers or companies. 
It's the case for all the times you used the word "user" when a feature is specific to a type of user.

**Review by EnzoGuillouche — CHANGES_REQUESTED** [review]
Reviewed for the functional specifications, they need changes to be merged.

**Comment by HiNett** [comment]
Closing the Pull Request, in accordance with the rules in place. One pull request per day, if not finalized by the end of the day/weekend, it will be closed.

**Pull request closed without merging**